src/pe-sp-service-outage-card/widget/pe-sp-service-outage-card.u-css.scss

```diff
@@ -0,0 +1,89 @@
+.x-pisn-sp-outcard {
+  position: relative;
+  max-width: 500px;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  text-rendering: optimizeLegibility;
+}
+
+.navigation {
+  position: absolute;
+  right: 0;
+  top: 0;
+
+  span.info {
+    vertical-align: bottom;
+  }
+
+  .btn {
+    padding-top: 2.5px;
+    padding-bottom: 2.5px;
+  }
+}
+
+.panel-heading {
+  .glyphicon {
+    color: white;
+  }
+
+  .add-info {
+    margin-top: 3px;
+  }
+}
+
+.btn:focus,
+.btn.focus,
+.btn:active:focus {
+  outline: none;
+  box-shadow: none;
+}
+
+.btn-link-danger {
+  color: $brand-danger !important;
+}
+
+.btn-link-success {
+  color: $brand-success !important;
+}
```

Comment:
use !important sparingly. use css specificity.
https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity

Reply:
Thanks! Totally agree, I was in a hurry 🙈